source/Calamari.CloudAccounts/AwsEnvironmentGeneration.cs

```diff
@@ -61,6 +62,7 @@ private AwsEnvironmentGeneration(ILog log, IVariables variables)
             assumeRoleArn = variables.Get("Octopus.Action.Aws.AssumedRoleArn")?.Trim();
             assumeRoleExternalId = variables.Get("Octopus.Action.Aws.AssumeRoleExternalId")?.Trim();
             assumeRoleSession = variables.Get("Octopus.Action.Aws.AssumedRoleSession")?.Trim();
+            assumeRoleDurationSeconds = variables.Get("Octopus.Action.Aws.AssumeRoleSessionDurationSeconds")?.Trim();
```

Comment:
I couldn't think of a better name for this variable (I would have called it the same thing).